docs/documenting/create-page.md

```diff
@@ -36,23 +48,18 @@ Content... Written in markdown.
 Additional keys for the file header:
 
 - `title`: This title should match with the name of the integration as written in the integration manifest file.
-- `ha_release`: The release when the integration was included, e.g., "0.38". If the current release is 0.37, make `ha_release` 0.38. If it's 0.30 or 0.40 please quote it with `' '`.
+- `ha_release`: The release when the integration was included. For example, "0.2025.9". If the current release is 2025.8, make `ha_release` 2025.9. If it end in a `0`, as in 2025.10, quote it with `' '`, otherwise the zero won't be displayed.
```

Comment:
```suggestion
- `ha_release`: The release when the integration was included. If the current release is 2025.8, make `ha_release` 2025.9. If it ends in a `0`, as in 2025.10, quote it with `' '`, otherwise the zero won't be displayed.
```
the example is already in the following sentence. If you prefer to keep it, remove the prefix `0.`